From ebe6e2cdde25e1a918d93a2c696b92bd1a2dfc6c Mon Sep 17 00:00:00 2001
From: Jeremy Bicha <jbicha@debian.org>
Date: Sun, 2 Dec 2018 14:51:23 -0500
Subject: [PATCH] Set HOME=$(CURDIR) to fix docs build

Unable to create local directories(/sbuild-nonexistent/.ansible/tmp):
 [Errno 13] Permission denied: '/sbuild-nonexistent'

Closes: #915316
---
 debian/rules | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/debian/rules b/debian/rules
index f6f8ad5a..098355e6 100755
--- a/debian/rules
+++ b/debian/rules
@@ -7,7 +7,7 @@ override_dh_clean:
 	dh_clean --exclude=test/units/inventory_test_data/group_vars/noparse/all.yml~
 
 override_dh_installdocs:
-	make docs
+	HOME=$(CURDIR) make docs
 	dh_installdocs
 
 #override_dh_auto_build:
-- 
2.19.1

